src/components/text/index.js

```diff
@@ -74,6 +75,7 @@ function createStyles() {
   return StyleSheet.create({
     container: {
       backgroundColor: 'transparent',
+      writingDirection: Constants.isRTL ? 'rtl' : 'auto'
```

Comment:
Here too. Worth checking just in case

Reply:
did check, Android is fine, ios works only after this change.

Reply:
ok, I doubled checked. on Android it wasn't fine, is it possible there's a difference between the public and private?